tests/test_home_page.py

```diff
@@ -1,23 +1,44 @@
+'''
+Checks functionality of HomePage
+'''
 from utilities.base_test_case import BaseTestCase
 from pages.home_page import HomePage
-import time
 
 
-class TestOpenHomePage(BaseTestCase):
+class TestHomePage(BaseTestCase):
+    '''
+    Class for HomePage tests
+    '''
 
-    def test_OpenHomePage(self):
+    def test_open_home_page(self):
         '''
         Verifies if the HomePage opens
         '''
-        assert self.driver.title == "Google"
+        home_page_title = "Google"
+        self.assertEqual(self.driver.title, home_page_title, "HomePage Failed to load")
 
-    def test_GoogleSearch(self):
+    def test_open_gmail_link(self):
+        '''
+        Verifies if Gmail login page opens
+        '''
+        home_page = HomePage(self.driver)
+        home_page.wait_for_element_to_load(HomePage.gmail_link)
+        home_page.get_gmail_link().click()
+
+        gmail_page_title = "Gmail: Private and secure email at no cost | Google Workspace"
+        self.assertEqual(self.driver.title, gmail_page_title, "Gmail link failed to load")
+
+    def test_google_search(self):
         '''
         Verifies if page with search results opens
         '''
         home_page = HomePage(self.driver)
```

Comment:
Since this is something that is called in all tests, it can be moved to setUp method or even setUpClass. (pay attention to inheritance to not simply override the parent setUp method, use super())